ui/desktop/README.md

```diff
@@ -176,6 +179,24 @@ Steps:
 
 TBD
 
+## Debug desktop client
+
+To debug the desktop client we recommend using node debugging client. 
```

Comment:
Could you possibly give a little more context on _when_ we would want to actually attach a debugger, maybe a scenario? I haven't come across the need yet

Reply:
@ZedLi Good Idea I will bring this topic to tech time.

That is correct, this instructions are just to debug the `main` process. I will update the documentation to reference that.

Absoulutley right, you do not need to use the `chrome` inspector. You can use different ones, as you mentioned you can use the one on webstorms. [More info about inspector clients here](https://nodejs.org/en/docs/guides/debugging-getting-started/#inspector-clients).

Reply:
I would also update the documentation with and scenario that shows the need to debug the `main` process.

I used the `main process` debug to help on the fix for use `env://` on the token. Not saying you need to the debug the main process to catch that bug, but it helped me.